modules/nodes-pods-configuring-replication.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-pods-configuring.adoc
+
+[id='nodes-pods-configuring-replication_{context}']
+= Controlling pod counts using replication controllers
+
+A replication controller ensures that a specified number of replicas of a pod are running at
+all times. If pods exit or are deleted, the replication controller acts to
+instantiate more up to the defined number. Likewise, if there are more running
+than desired, it deletes as many as necessary to match the defined amount.
+
+A replication controller configuration consists of:
+
+* The number of replicas desired (which can be adjusted at runtime).
+* A pod definition to use when creating a replicated pod.
+* A selector for identifying managed pods.
+
+A selector is a set of labels assigned to
+the pods that are managed by the replication controller. These labels are
+included in the pod definition that the replication controller instantiates.
+The replication controller uses the selector to determine how many
+instances of the pod are already running in order to adjust as needed.
+
+The replication controller does not perform auto-scaling based on load or
+traffic, as it does not track either. Rather, this would require its replica
+count to be adjusted by an external auto-scaler.
+
+A replication controller is a core Kubernetes object called `*ReplicationController*`.
```

Comment:
wrong format

Reply:
@kalexand-rh Thanks. This was straight from the source. The docs seem inconsistent on formatting for RCs. Most references have no mark-up. I will change to that.

Reply:
I think that's the best interpretation of the 4.0 markup guidelines.